README.md

```diff
@@ -242,4 +243,4 @@ Make sure you have [Node.js](http://nodejs.org/) installed.
 
 ## Licence
 
-My work is released under the [MIT license](https://raw.githubusercontent.com/TimvanScherpenzeel/detect-features/master/LICENSE).
\ No newline at end of file
+My work is released under the [MIT license](https://raw.githubusercontent.com/TimvanScherpenzeel/detect-features/master/LICENSE).
```

Comment:
my editor seems to have just edited trailing whitespace. let me know if you want me to undo this.

Reply:
No problem, I'll quickly fix it after merge.